src/ray/raylet/node_manager.cc

```diff
@@ -1853,47 +1853,45 @@ void NodeManager::ProcessWaitRequestMessage(
     } else {
       // We failed to write to the client, so disconnect the client.
       std::ostringstream stream;
-      stream << "Failed to write WaitReply to the client. Status " << status
-             << ", message: " << status.message();
+      stream << "Failed to write WaitReply to the client. Status " << status;
```

Comment:
I think we should call `ToString` here, for `ostream`, you first format string into stream's internal buffer, then copy it out to a new string (one format + one string copy + string resize vs one format)

Reply:
Discussed offline, perf is not a concern here.

Reply:
^ The above is nit, non merge blocker; if prompt message is needed we need to use stringstream (or expose a mutable function from `Status`)

Another nit (also non merge blocker) I have is, it's better to `std::move(os).str()`, so after we upgrade to C++20 we get performance gain with no code change.